<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QMainWindow" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>175</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>175</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>175</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>frmMain</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>302</width>
      <height>143</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLineEdit" name="edtUdpPort">
     <property name="geometry">
      <rect>
       <x>221</x>
       <y>22</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>8091</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSend">
     <property name="geometry">
      <rect>
       <x>208</x>
       <y>108</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPort">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>23</y>
       <width>29</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chbAutoSend">
     <property name="geometry">
      <rect>
       <x>184</x>
       <y>46</y>
       <width>51</width>
       <height>18</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Auto:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtPhone">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>80</y>
       <width>263</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0677895602</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblIP">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>21</y>
       <width>19</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IP:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtIP">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>125</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtInterval">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>48</y>
       <width>55</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblInterval">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>50</y>
       <width>79</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Interval (us):</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
